src/System.ServiceModel.Http/src/System.ServiceModel.Http.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>System.ServiceModel.Http</AssemblyName>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
That's OK.  Full facade is just a special case of partial facade that is all type forwards.